app/src/main/java/org/p2p/wallet/history/model/rpc/RpcHistoryTransaction.kt

```diff
@@ -378,6 +379,7 @@ sealed class RpcHistoryTransaction(
         val tokenSymbol: String,
         val iconUrl: String?,
         val fees: List<RpcFee>?,
+        val message: String
```

Comment:
why not bridgeServiceKey?

Reply:
because in transaction it is `message` and in SendDetails it is `id`
third option will just make more confuse =(